library/spdm_crypt_lib/libspdm_crypt_crypt.c

```diff
@@ -3336,10 +3308,6 @@ bool libspdm_req_asym_sign_hash(
         /* Need use SPDM 1.2 signing*/
 
         switch (req_base_asym_alg) {
-        case SPDM_ALGORITHMS_BASE_ASYM_ALGO_TPM_ALG_SM2_ECC_SM2_P256:
-            param = "";
```

Comment:
@steven-bellock , have you verified the different between NULL and "" in SM2?

Reply:
`param` would map to the `id_a` parameter in that function. But I see what you mean. Let me try this a different way.